roles/fips/tasks/main.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: 'Install dracut-fips'
+  package: 
+    name: dracut-fips
+    state: present
+
+- name: 'Run dracut to rebuild initramfs'
+  command: dracut --force
+
+- name: 'Edit kernel command-line to include the fips=1 argument'
+  command: grubby --update-kernel=DEFAULT --args=fips=1
+
+- name: 'Get uuid'
+  command: 'findmnt -no uuid /boot'
+  register: result
+
+- name: 'Edit kernel command-line to include the boot=UUID=XXXX or boot=LABEL=XXXX or boot=/dev/DEVICE argument'
+  command: "grubby --update-kernel=DEFAULT --args=boot=UUID={{ result.stdout }}"
+
+- name: reboot vm
+  shell: sleep 2 && shutdown -r +1 "Ansible reboot"
+  async: 0
+  poll: 0
+  ignore_errors: true
+
+- name: Waiting for reboot
+  local_action: wait_for host={{ ansible_ssh_host }} state=started
+
+- name: 'Verify FIPS enabled - 1 : Enabled ; 0 : Disabled '
```

Comment:
Perhaps this needs to be compared and the role failed if it doesn't match. When you enable multiple roles this will be easy to miss.